### PR TITLE
Made the button corners rounded 

### PR DESCRIPTION
**Pull request by Arshilgenius**
Fixes issue #784 
Changes: Made the button corners rounded  for Login   sign up and forgot password activity in both portrait and landscape mode.

Screenshots for the change:

### CODE REVIEW

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
         android:layout_width="250dp"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/margin_medium"
-        android:background="@color/md_grey_300"
+        android:background="@drawable/rounded_button"
```
Comment (mayank408) [inline app/src/main/res/layout/activity_forgot_password.xml]: Do for the landscape as well.

File: app/src/main/res/layout/activity_sign_up.xml [hunk 2]
```diff
@@ -120,7 +120,7 @@
             android:layout_width="250dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
-            android:background="@color/md_grey_300"
+            android:background="@drawable/rounded_button"
```
Comment (mayank408) [inline app/src/main/res/layout/activity_sign_up.xml]: here too

File: app/src/main/res/layout/activity_forgot_password.xml (hunk 1 above)
Comment (Arshilgenius) [inline app/src/main/res/layout/activity_forgot_password.xml]: I think i did that too, but forgot to attach images.
Can u please check? 
thanks.

Comment (mayank408) [inline app/src/main/res/layout/activity_forgot_password.xml]: Have you done this for login?

Comment (Arshilgenius) [inline app/src/main/res/layout/activity_forgot_password.xml]: No, Should i do that? it looks like this.

screen shot 2017-07-03 at 2 18 07 pm

Comment (mayank408) [inline app/src/main/res/layout/activity_forgot_password.xml]: okk great 👍

Comment (Arshilgenius) [inline app/src/main/res/layout/activity_forgot_password.xml]: Do i have to create another PR for this commit (changing the login button) ?

**Comment by mayank408** [comment]
@Arshilgenius Pls squash more commits. Rest all is good. Cheers 👍

**Comment by Arshilgenius** [comment]
@mayank408  Can you tell me what should i do now?

File: app/src/main/res/layout/activity_forgot_password.xml (hunk 1 above)
Comment (Arshilgenius) [inline app/src/main/res/layout/activity_forgot_password.xml]: Can u please merge my changes?

Comment (Arshilgenius) [inline app/src/main/res/layout/activity_forgot_password.xml]: I don't know how to squash as i am new to contributing to open source.

**Comment by chiragw15** [comment]
@Arshilgenius Can you reduce size of button in landscape mode of login screen.


Also, in landscape mode of signup and forgot password activity, center align radio buttons and url edit text

**Comment by chiragw15** [comment]
And follow this for squashing commits http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by chashmeetsingh** [comment]
Also, please change the placeholder color and radio buttons color in Forgot Password Activity from green to white and radio button color from black to white in the SignUp activity.

**Comment by mayank408** [comment]
@chashmeetsingh No need to change the placeholder color they will already be changed in the theme PR.

**Comment by Arshilgenius** [comment]
I have done the changes, please review and suggest any changes

**Comment by chiragw15** [comment]
Left align radio buttons with password input Edit Text in signup and login activity (landscape mode). Also, in login activity(landscape mode) decrease width of login button.

**Comment by Arshilgenius** [comment]
Are these okay?

**Comment by chiragw15** [comment]
First two are good. In third one, keep the height same as in the below screenshot and reduce width so that it is slightly less than "Forgot password?" textview. Maybe start from first 'o' in "forgot" to 'd' in "password".

**Comment by chiragw15** [comment]
And also resolve conlicts. Do you know how to do that?

**Comment by Arshilgenius** [comment]
Okay i will do that, and i don't know how to resolve conflicts. Please help

**Comment by Arshilgenius** [comment]
how's this?
Also i am getting confused with

**Comment by chiragw15** [comment]
Yup, fine. Just commit and push your changes now.
